frontend/messages/en.json

```
@@ -1125,6 +1125,11 @@
 	"importVersionNotCompatibleWithCurrentVersion": "The import version is not compatible with the current version of the application",
 	"errorOccuredDuringImport": "An error occurred during the import",
 	"successfullyImportedFolder": "Folder successfully imported",
+	"missingLibrariesInImport": "Some libraries are missing, see the list above",
```

Biome (1.9.4)

[error] The key missingLibrariesInImport was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)